package.json

```diff
@@ -136,7 +136,7 @@
     "grunt": "~1.0.1",
     "grunt-cli": "~1.2.0",
     "highlight.js": "^9.15.8",
-    "html-webpack-plugin": "^3.2.0",
+    "html-webpack-plugin": "^4.3.0",
```

Comment:
I don't think we use this anywhere

Reply:
It is used in the widgets and probably wasn't removed when we split the widgets out of the sdk. Feel free to remove it.